packages/reports/tests/acceptance/navi-report-test.js

```diff
@@ -1238,26 +1238,25 @@ module('Acceptance | Navi Report', function(hooks) {
   });
 
   test('Updating chart series', async function(assert) {
-    assert.expect(3);
+    assert.expect(4);
```

Comment:
Do we need this test anymore? It is now just testing the c3 feature, rather than something we built.

Reply:
That's true. I don't mind either way. Could help as we upgrade c3 though to make sure a series selector feature exists? Though if the team thinks we should remove, I'd be happy to

Reply:
I feel like it's good to keep it so we make sure the functionality is there in Navi.